docker-compose.yml

```diff
@@ -23,6 +23,9 @@ services:
     environment:
       VIRTUAL_HOST: cms.prx.docker
       LOCAL_ENV: "true"
+      FEEDER_DB_HOST: pg
```

Comment:
Are these really required here?  I thought they were only used by the feeder import script.  But maybe the tests need them or something.

Reply:
Yep,  the tests in docker wont run unless these are set.

Reply:
+1